week0003/thainguyentran/CSharp/week0003.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Linq;
+
+namespace DVRK
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Dictionary<string, double> clients = new Dictionary<string, double>();
+            string[] temp = Console.ReadLine().Split(' ');
+            switch(Convert.ToChar(temp[0]))
+            {
+                case 'A':
```

Comment:
Make cases more readable. You could create constants like `ACTION_ADD_CLIENT = 'A'`